.github/workflows/ci.yml

```
@@ -16,6 +16,11 @@ jobs:
 
     steps:
     - uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8
+
+    - name: Check if any local image is used in docker-compose.yml
+      run: |
+        if [[ "$(egrep -w 'image: repository-service-tuf-api|image: repository-service-tuf-worker|image: repository-service-tuf-cli' docker-compose.yml -c)" -ne "0" ]]; then echo "Found" && exit 1; fi
```

```suggestion
        egrep -w 'image:\s+repository-service-tuf-api|image:\s+repository-service-tuf-worker' docker-compose.yml
        if [[ $rc -eq "0" ]]; then \
          echo "Local image has been used in docker-compose.yml" \
          exit 1 \
        fi
```

- The `repository-service-cli` is not provided as a image and can be removed
- I added some extra in the regex to avoid more than one white space
- Instead of using count, we can use the shell return code (`$?`), which is the standard.
- neat: multiline for better reading